### PR TITLE
MS total exemptions

### PR DESCRIPTION
**Pull request by ZiqianCui**
Added total exemptions, waiting for the dependents_exemption to be added

### CODE REVIEW

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: run the `git pull upstream master`

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Mississippi total exemptions & related missing fils (calibration -> populations, programs(medicaid, snap))
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Mississippi total exemptions.
```

File: policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_total_exemptions.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class ms_total_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Mississippi total exemption allowance"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MS
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_total_exemptions.py:10]: reference = ""

File: policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_total_exemptions.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
+from policyengine_us.model_api import *
+
+
+class ms_total_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Mississippi total exemption allowance"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_total_exemptions.py:7]:
```suggestion
    label = "Mississippi total exemptions"
```

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**